packages/emoji-mart/src/components/Picker/Picker.tsx

```diff
@@ -1128,16 +1134,17 @@ export default class Picker extends Component {
         )}
 
         <div ref={this.refs.scroll} class="scroll flex-grow padding-lr">
-          <div
+          <section
             style={{
               width: this.props.dynamicWidth ? '100%' : lineWidth,
               height: '100%',
             }}
+            aria-label="Available emojis"
```

Comment:
Emoji mart uses i18n, please add it to the translations: https://github.com/slidoapp/emoji-mart/blob/main/packages/emoji-mart-data/i18n/en.json

Reply:
Ah yeah I forgot, thanks!